components/shopify/actions/add-products-to-custom-collections/add-products-to-custom-collections.mjs

```diff
@@ -0,0 +1,47 @@
+import shopify from "../../shopify.app.mjs";
+
+export default {
+  key: "shopify-add-products-to-custom-collections",
```

Comment:
Is it really needed to change the component key here, rather than just the name and description? Seeing as it's just an added `s` and, correct me if I'm wrong, I believe that would require any users using it to remove the step from their existing workflows and re-configure it with the new action.

I'm not sure where we draw the line between keeping the component key 100% faithful to its name, and avoiding this potential hassle for users that already have the component configured in their workflows.

BTW, if the key is indeed updated, I think the version should be `0.0.1`